share/completions/blender.fish

```diff
@@ -0,0 +1,124 @@
+function __blender_list_scenes -a file
+    blender --background $file --python-expr 'import bpy
+
+for name in [scene.name for scene in list(bpy.data.scenes)]:
+    print(f"\t{name}")' |
+        string replace -r -f '^\s+' ''
+end
+
+function __blender_list_addons
+    path basename /usr/share/blender/scripts/addons/*.py | path change-extension ''
+end
+
+function __blender_list_engines
+    blender --background --engine help | string replace -r -f '^\s+' ''
+end
+
+function __blender_echo_input_file_name
+    echo $argv |
+        string split -n ' ' |
+        string match -r -v '^-' |
+        head --lines=1
+end
+
+function __blender_complete_addon_list
+    set -l previous_token (commandline -oc)[-1]
+    set -l current_token (commandline -t)
+
+    if test $previous_token = --addons
```

Comment:
This still needs to be quoted.

Reply:
I am confused: I see my quotation fix [there](https://github.com/EmilySeville7cfg/fish-shell/commit/5745c7bc1ad2f93f9f99661dbd8544b187e1568e) but not [here](https://github.com/fish-shell/fish-shell/pull/9905/files#diff-886e83748a6c40680df274dcd2694c8ad5ca4383bad2ec7a1553ea070963730eR28).

Reply:
![image](https://github.com/fish-shell/fish-shell/assets/42812113/1559287a-a317-435b-bc16-5528392fc7f1)

I feel something went wrong: in a web editor I see quotes around $previous_token, while in your comment - not.